weed/server/filer_grpc_server_rename.go

```diff
@@ -34,6 +34,20 @@ func (fs *FilerServer) AtomicRenameEntry(ctx context.Context, req *filer_pb.Atom
 		return nil, fmt.Errorf("%s/%s not found: %v", req.OldDirectory, req.OldName, err)
 	}
 
+	if oldEntry.IsDirectory() {
```

Comment:
these logic is different from what the PR described?

Reply:
I'm sorry that I forgot to describe it in the PR.
This logic is the same as the move operation of mount, which is used to limit the directory move to non-empty directories and files.